src/screens/ReportCorrectionFormScreen/sections/ElevatorCorrectionSection.tsx

```
@@ -0,0 +1,186 @@
+  return (
+    <Container>
+      <SectionTitle>엘리베이터 정보</SectionTitle>
+
+      <SubLabel>엘리베이터</SubLabel>
+      <OptionRow>
+        {[
+          {value: true, label: '있음'},
+          {value: false, label: '없음'},
+        ].map(option => {
+          const isSelected =
+            option.value === true ? hasElevator : !hasElevator;
```

ESLint

[error] Delete `⏎···········`

(prettier/prettier)